Tests/RegressionTests.cs

```diff
@@ -77,7 +77,8 @@ private static TestCaseData[] GetRegressionTestParameters()
             var nonDefaultStatuses = new Dictionary<string, AlgorithmStatus>
             {
                 {"TrainingInitializeRegressionAlgorithm", AlgorithmStatus.RuntimeError},
-                {"OnOrderEventExceptionRegression", AlgorithmStatus.RuntimeError}
+                {"OnOrderEventExceptionRegression", AlgorithmStatus.RuntimeError},
+                {"WarmUpAfterIntializeRegression", AlgorithmStatus.RuntimeError }
```

Comment:
A separate issue, but starting to think this end state should probably be owned by the algorithm itself, not this 3rd party 🤔

Reply:
I could see that, maybe we should consider extending the IRegressionAlgorithmDefinition interface to contain a member for expected status?